### ISSUE TITLE
Locale and time zone can be configured in settings.yml

### ISSUE THREAD

**Issue by pmackay** [codeforamerica/ohana-api#322]
Could these values defined in `application.rb` be defined in `settings.yml` instead? 

I could do a PR.

BTW, I'm assuming its ok to raise feature request issues and then do a PR?

**Comment by monfresh** [comment]
Yes, and yes, but since this is a feature request, please reword the issue title so that it reads like a request or a question, such as `Allow locale and time zone to be configured in settings.yml`. Otherwise, it sounds like locale and time zone can already be configured in settings.yml, and that the fact that they can be configured there is a bug.

**Comment by pmackay** [comment]
Sure. I tend to name issues for features/stories as if they describe the desired state, as it were. Noted :)
